string sample_token
float64[] translation
float64[] size
float64[] rotation
float64[] velocity
string detection_name
float64 detection_score
